### PR TITLE
Exit the process on a TF runtime error from the TF_SessionRun() call, instead of check failing.

### PR DESCRIPTION
**Pull request by mhong**

### CODE REVIEW

File: stdlib/public/TensorFlow/CompilerRuntime.swift [hunk 1]
```diff
@@ -547,7 +550,10 @@ extension TFState {
       targetNodeSpecs, Int32(targetNodeSpecs.count),
       /*run_metadata*/nil, status
     )
-    checkOk(status)
+    if (TF_GetCode(status) != TF_OK) {
+      logToStderr(String(cString: TF_Message(status)))
```
Comment (rxwei) [inline stdlib/public/TensorFlow/CompilerRuntime.swift]: It doesn't matter much, but calling `fputs` is easier and more efficient.

```swift
_ = fputs(TF_Message(status), stderr)
```

File: stdlib/public/TensorFlow/CompilerRuntime.swift [hunk 2]
```diff
@@ -799,7 +808,7 @@ extension _TensorComputation {
 // user code, so they are generally just wrappers around the implementation
 // above.
 
-/// Loads the TF computation from a binary TF FunctionDef proto given by 'bytes'
+/// Load the TF computation from a binary TF FunctionDef proto given by 'bytes'
```
Comment (rxwei) [inline stdlib/public/TensorFlow/CompilerRuntime.swift]: Plural verbs in comments were intentional - Swift code usually uses plural verbs. If you see other parts of the `TensorFlow` module and the standard library, they are plurals too.

File: stdlib/public/TensorFlow/CompilerRuntime.swift [hunk 3]
```diff
@@ -87,7 +87,8 @@ public enum _RuntimeConfig {
 
   /// When true, run the entire tensor computation in
   /// _TFCStartTensorComputation(), instead of running it on a separate thread.
-  /// - Note: Set to true only for debugging purposes.
+  /// - Note: Set to true only for debugging purposes, as it has limited
+  /// functionality (e.g. no sends/recvs support).
```
Comment (rxwei) [inline stdlib/public/TensorFlow/CompilerRuntime.swift]: Start of a new line in a doc comment item should align with the name of that item, i.e. "Note".

File: stdlib/public/TensorFlow/CompilerRuntime.swift [hunk 4]
```diff
@@ -466,6 +467,8 @@ extension TFState {
     inputTensors.append(cTensor!)
   }
 
+  /// Run the tensor program. Abort the process on error, and emit an error
```
Comment (rxwei) [inline stdlib/public/TensorFlow/CompilerRuntime.swift]: Same here regarding plural verbs.

File: stdlib/public/TensorFlow/CompilerRuntime.swift (hunk 1 above)
Comment (mhong) [reply stdlib/public/TensorFlow/CompilerRuntime.swift]: Done.

**Review by rxwei — APPROVED** [review]
LGTM!

**Review by lattner — APPROVED** [review]

**Comment by mhong** [comment]
CI passed: https://ci-external.swift.org/view/Pull%20Request/job/swift-PR-TensorFlow-Linux/468/

**Pull request merged**